integration_tests/models/marts/exposures.yml

```diff
@@ -13,6 +13,7 @@ exposures:
     
     depends_on:
       - ref('fct_model_6')
+      - ref('fct_model_9')
```

Comment:
Why was this added?

Reply:
i don't remember! will remove and see if i break anything

Reply:
ah, I had to materialize `dim_model_7` for the metrics I made to work, so I added this here to make sure we still had an ephemeral model feeding the exposure for this check